.github/workflows/ci-tests.yml

```
@@ -21,7 +23,7 @@ jobs:
       matrix:
         os:
           - ubuntu-22.04
```

```suggestion
          - ubuntu-20.04
```

do we want to match this with what we have in [publish.yml](https://github.com/subspace/subspace-desktop/pull/304/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R16)

---

I believe we don't. It was like that in monorepo as well. I guess for testing, we are aiming for the latest technology, but for production, the most compatible one. That was my take